main/store/actions/index.js

```diff
@@ -350,6 +351,12 @@ module.exports = {
             main.origins[origin].chain = updatedNetwork
           }
         })
+
+        main.networksMeta[updatedNetwork.type][updatedNetwork.id] = main.networksMeta[updatedNetwork.type][updatedNetwork.id] || {}
```

Comment:
can this be cleaned up by assigning the meta to a variable and updating that rather than using bracket notation access over and over?

Reply:
Yeah, I was thinking we probably want to remove this all together as it was added because the symbol seems to be stored in multiple places now.

Reply:
noticed that too. we should decide if we want to store the symbol as part of the chain as part of the chain's native currency